app/upgrades/v1/mainnet/finality_params.go

```diff
@@ -1,13 +1,11 @@
 package mainnet
 
-// TODO Some default parameters. Consider how to switch those depending on network:
-// mainnet, testnet, devnet etc.
 const FinalityParamStr = `{
-  "max_active_finality_providers": 100,
-  "signed_blocks_window": 100,
+  "max_active_finality_providers": 60,
+  "signed_blocks_window": 10000,
   "finality_sig_timeout": 3,
-  "min_signed_per_window": "0.1",
-  "min_pub_rand": 100,
-  "jail_duration": "86400s",
-  "finality_activation_height": 17500
+  "min_signed_per_window": "0.3",
```

Comment:
We should consider decreasing this to 0.2. cc @babylonlabs-io/core-dev

Reply:
updated to 0.2